chiabot.dart

```diff
@@ -83,7 +83,7 @@ main(List<String> args) async {
 
       log.info("Generating new report #${counter}");
 
-      cache.init(config.parseLogs, config.userNumber);
```

Comment:
This `config.userNumber` argument is too late in the process for it to be any use.

Reply:
This would break some features, it's not as easy as it seems.
Config depends on Cache's binPath (which stores where chia binaries are, or where they were defined in the case that the user git cloned chia's repo)
Cache depends on config's parseLogs and userNumber, that's why I "had" to initialize it twice as a quick fix.
Not doing this would not be able to get the state of parseLogs, or userNumber defined in config.json
A better way to fix this would be to show the initial screen (with ids and links) as a separate class from config and cache.
This could be shown after cache.init in this line.

Reply:
Your commit below would fix the issue with userNumber but would break parseLogs. I'm not sure if I am explaining this clearly.

Reply:
Oh my bad. I thought you had deleted the whole cache.init line. Turns out it was the config.userNumber argument. Ok seems like you got it haha! Well done.